### DISCUSSION TITLE
Import syntax with v6 and TypeScript?

### DISCUSSION THREAD

**Discussion by aaronadamsCA** [Automattic/mongoose#11040] [Q&A] [answered]
I see this:

> https://mongoosejs.com/docs/faq.html#destructured-imports
> 
> The only import syntax Mongoose supports is `import mongoose from 'mongoose'`.

And then I see this:

> https://mongoosejs.com/docs/typescript.html
> 
> ```ts
> import { Schema, model, connect } from 'mongoose';
> ```

And I'm wondering if someone can help me out.

Did this limitation get fixed in v6? I know I've been bit by it before, even in TypeScript, but now I see TypeScript examples using non-default imports even in the docs, so I'm not sure what to think.

**Comment by vkarpov15** [comment, marked as answer]
Those docs are out of date, we removed that entry from our FAQ. We now support `import { Schema } from 'mongoose'`.
